src/object.c

```diff
@@ -86,7 +86,9 @@ int git_object__from_raw(
 	GIT_ERROR_CHECK_ALLOC(object);
 	object->cached.flags = GIT_CACHE_STORE_PARSED;
 	object->cached.type = type;
-	git_odb_hash(&object->cached.oid, data, size, type);
+	if ((error = git_odb_hash(&object->cached.oid, data, size, type)) < 0) {
+		return error;
+	}
```

Comment:
No need for braces around single-line bodies :)

Reply:
Argh, amending code is hard.